.doc_gen/readmes/services.yml

```diff
@@ -0,0 +1,181 @@
+actions:
+  cloudwatch:
+  - "[Create an alarm](./cw-ruby-example-create-alarm.rb)"
```

Comment:
Template is to include the wire API action after the link, so:
```
[Create an alarm](./cw-ruby-example-create-alarm.rb) (CreateAlarm)
```

Reply:
These did not exist before so it may take me a minute.